includes/entities/class-fs-plugin-plan.php

```diff
@@ -88,6 +88,10 @@ class FS_Plugin_Plan extends FS_Entity {
 		 * @var bool Is featured plan.
 		 */
 		public $is_featured;
+        /**
+         * @var bool Is hidden plan.
+         */
+        public $is_hidden;
```

Comment:
@DanieleAlessandra since the rest of the file is indented with tab, let's also use tab here.